q2_diversity/plugin_setup.py

```diff
@@ -18,18 +20,68 @@
     package='q2_diversity'
 )
 
-# TODO create decorator for promoting functions to methods. This info would
-# be moved to the decorator calls.
 plugin.methods.register_function(
-    function=q2_diversity.beta_diversity,
+    function=q2_diversity.beta_diversity_phylogenetic,
     # TODO require a uniform sampling effort FeatureTable when predicates exist
     inputs={'feature_table': FeatureTable[Frequency],
-            # TODO this is optional; how do we handle that here?
             'phylogeny': Phylogeny},
     parameters={'metric': Str},
```

Comment:
Would you mind adding `Str % Choices({...})`? it should be importable now from `qiime.plugin`